### PR TITLE
Theatre 45 slider

### PR DESCRIPTION
**Pull request by Kate-Kryvopustova**

### CODE REVIEW

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  align-items: center;
+  cursor: pointer;
+  position: absolute;
+  bottom: 0;
+  width: 70px;
+  z-index: 1;
+  -webkit-transition: all 0.3s ease;;
+  -moz-transition: all 0.3s ease;;
+  -o-transition: all 0.3s ease;;
+  transition: all 0.3s ease;
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss:66]: Use mixins

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  color: #b49864;
+  text-decoration: none;
+}
+
+.title {
+  height: 60px;
+  font-family: Montserrat;
+  font-size: 40px;
+  font-stretch: normal;
+  font-style: normal;
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss:18]: Use mixins

File: src/app/components/partials/actors-slider/actors-slider.component.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        [style.background]="'url(' + item.avatar.employee_big.url + ')'"
+        [routerLink]="['', 'persons', item.slug]"
+      >
+      </div>
+    </ngu-tile>
+    <ul class="myPoint" NguCarouselPoint>
+      <li
+        *ngFor="let j of myCarousel.pointNumbers;
+        let i = index"
+        [class.active]="j===myCarousel.activePoint"
```
[the remark this answers is not in the document]
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.html]: Done.

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 4]
```diff
@@ -0,0 +1,135 @@
+.wrapped {
+  margin: 0;
+  height: 897px;
```
[the remark this answers is not in the document]
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.scss]: Done.

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  white-space: nowrap;
+  overflow: auto;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.myPoint  li {
+  background-color: #868583;
```
[the remark this answers is not in the document]
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.scss]: Done.

File: src/app/components/partials/actors-slider/actors-slider.component.scss (hunk 1 above)
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.scss]: Done.

File: src/app/components/partials/actors-slider/actors-slider.component.scss (hunk 2 above)
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.scss:47]: Done.

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 6]
```diff
@@ -1,18 +1,47 @@
...
+
+@mixin opacity {
+  -webkit-transition: all 0.3s ease;;
+  -moz-transition: all 0.3s ease;;
+  -o-transition: all 0.3s ease;;
+  transition: all 0.3s ease;
+  opacity: 1;
+}
+
+@mixin hover_opacity {
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss:14]: `-`

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 7]
```diff
@@ -1,18 +1,47 @@
...
+
+@mixin gradient {
+  background: linear-gradient(rgba(14, 12, 8, 0), rgba(14, 12, 8, 0.5), rgba(14, 12, 8, 0.75), #0e0c08);
+}
+
 .wrapped {
   margin: 0;
-  height: 897px;
+  height: 900px;
   padding: 83px 0 86px;
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss:33]: 83? 86?

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 8]
```diff
@@ -1,18 +1,47 @@
...
   text-decoration: none;
 }
 
 .title {
   height: 60px;
-  font-family: Montserrat;
+  font-family: 'Montserrat', sans-serif;
   font-size: 40px;
   font-stretch: normal;
   font-style: normal;
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss:47]: Delete 
`
  font-stretch: normal;
  font-style: normal;
`

File: src/app/components/partials/actors-slider/actors-slider.component.scss (hunk 2 above)
Comment (Serhio77777) [reply src/app/components/partials/actors-slider/actors-slider.component.scss:47]: No...

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 9]
```diff
@@ -42,8 +70,8 @@ a {
   margin: 32px;
   min-height: 200px;
   background-size: cover !important;
-  width: 352px;
-  height: 528px;
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss]: mixins

File: src/app/components/partials/actors-slider/actors-slider.component.scss [hunk 10]
```diff
@@ -52,53 +80,37 @@ a {
...
-  opacity: 0.6;
+  @include hover_opacity;
 }
 
 .right {
-  display: flex;
-  justify-content: center;
+  @include display;
+  @include opacity;
+  @include gradient;
```
Comment (Serhio77777) [inline src/app/components/partials/actors-slider/actors-slider.component.scss:102]: Make it for reusable

**Review by Serhio77777 — CHANGES_REQUESTED** [review]

File: src/app/components/partials/actors-slider/actors-slider.component.scss (hunk 7 above)
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.scss:6]: Done

File: src/app/components/partials/actors-slider/actors-slider.component.scss (hunk 10 above)
Comment (Kate-Kryvopustova) [reply src/app/components/partials/actors-slider/actors-slider.component.scss:64]: Done

File: src/app/components/home-page/home-page.component.ts [hunk 11]
```diff
@@ -1,36 +1,44 @@
...
+  limit = '3';
+  listPost: Array<NewsItem> = [];
+  readonly canonicalUrl = environment.canonicalUrl;
+
+  constructor(private gatewayService: GatewayService,
+              private loaderService: LoaderService) { }
+
+  ngOnInit() {
+    this.loaderService.start('home');
+    this.gatewayService.getNews(this.limit, 1, 'uk').subscribe((res: { posts }) => {
```
Comment (yana-tsylinska) [inline src/app/components/home-page/home-page.component.ts:26]: use localeId everywhere, not hardcoded locale

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]

File: src/app/components/home-page/home-page.component.ts (hunk 11 above)
Comment (NataliyaKoval) [reply src/app/components/home-page/home-page.component.ts]: done

**Review by yana-tsylinska — APPROVED** [review]

**Pull request merged**